drafttopic/utilities/fetch_wikiprojects.py

```diff
@@ -61,17 +61,13 @@ def main(argv=None):
         r'\[\[Wikipedia:WikiProject Council/Directory/([A-Za-z_, ]+)\|([A-Za-z_, ]+)\]\]='
 WPListingRegex =\
         r'See the full listing \[\[Wikipedia:WikiProject Council/Directory/([A-Za-z_,/ ]+)'
```

Comment:
"Wikipedia:WikiProject Council/Directory" should be a parameter or a static variable

Reply:
fixed. Only left to figure out how to parameterize the complex regex - wp_section_regex